src/main.rs

```diff
@@ -1,10 +1,12 @@
 // SPDX-FileCopyrightText: 2022 Profian Inc. <opensource@profian.com>
 // SPDX-License-Identifier: AGPL-3.0-only
 
-use std::fs::File;
+use std::error::Error;
```

Comment:
If you're fixing the clippy lint, then also please import this via the full path (`std::error::Error`) at call site, which is the convention used currently throughout the code base to disambiguate (e.g. `anyhow::Error`)

Reply:
Sure, I can do that quickly.